.travis.yml

```diff
@@ -17,18 +17,26 @@ matrix:
       env:
         - OS=ubuntu-14.04
         - JOBQUEUE=sge
+    - python: "2.7"
+      env:
+        #For launching only basic unit tests
+        - OS=ubuntu-14.04
+        - JOBQUEUE=none
+    - python: "3.6"
+      env:
+        #For launching only basic unit tests
+        - OS=ubuntu-14.04
+        - JOBQUEUE=none
 env:
   global:
     - DOCKER_COMPOSE_VERSION=1.6.0
 
 before_install:
   - pwd
-  - docker version
-  - docker-compose version
-  # Install miniconda
+  # Install miniconda (only needed for basic unit tests)
   - ./ci/conda_setup.sh
   - export PATH="$HOME/miniconda/bin:$PATH"
-  - conda install --yes -c conda-forge python=$TRAVIS_PYTHON_VERSION dask distributed flake8
+  - conda install --yes -c conda-forge python=$TRAVIS_PYTHON_VERSION dask distributed flake8 pytest docrep
```

Comment:
This can be done in another PR but as you mentioned before, you may not need conda at all for `JOBQUEUE=sge`, so you could actually do this:
* move miniconda setup + conda install to none.sh `jobqueue_before_install`
* move flake8 to none.sh `jobqueue_script` or maybe leave it in .travis.yml  but only run it when `[[ "$JOBQUEUE" == "none" ]].  IIRC Travis will run all the commands in the `script` stage even if one command fails which is handy because you generally want to have all the failures at once.

Reply:
I'm still not sure about that. I know this is useless, but I like the idea of the flake8 checks running all the time... Don't know why.
But installing conda only for this while these checks are done on the "none" environement seems useless. Another PR?

Reply:
Finally when writing the above comment, I decided to implement it here, because keeping the conda install and flake8 check in every build was nonsense.